pkg/notification/adaptor/controller/notification.ts

```diff
@@ -26,4 +49,137 @@ export class NotificationController {
 
     return Result.ok(undefined);
   }
+
+  async fetchNotifications(
+    actorID: string,
+    limit?: number,
+    afterID?: string,
+    beforeID?: string,
```

Comment:
```suggestion
    options: {
      limit?: number;
      afterID?: string;
      beforeID?: string;
    } = {},
```
位置引数である `afterID` と `beforeID` を同時に設定すべきでないのは, このメソッドを利用するうえで `beforeID` だけを設定するのに少し煩わしい書き方が必要になったりとあまり好ましく無さそうです. オブジェクトによる指定を検討してみてください.